content-repo/extra-docs/articles/Email-Communication-Readme.md

```diff
@@ -0,0 +1,174 @@
+---
+id: email-communication
+title: Email Communication
+description: Communication across and between departments is a vital component of collecting information, and managing and remediating security events. The Email Communication content pack enables security teams to automate and streamline the communication and notification process with users across your organization via email.
+ 
+---
+This Email Communication pack enables security teams to reply to an email as part of an incident. The SOC team is able to communicate directly through the Cortex XSOAR platform as part of the remediation process. 
+
+
+
+
+## Pack Workflow
+When an email is sent to the email address configured in your email integration, your email listener fetches the incoming email in Cortex XSOAR. 
```

Comment:
Need to change the word hash to number because we're generating a random number and not creating a hash.